packages/cli/src/commands/storybook.js

```diff
@@ -43,24 +47,30 @@ export const builder = (yargs) => {
       type: 'boolean',
       default: false,
     })
-    .check((argv) => {
-      if (argv.build && argv.smokeTest) {
-        throw new Error('Can not provide both "--build" and "--smoke-test"')
-      }
-      if (argv.build && argv.open) {
-        throw new Error('Can not provide both "--build" or "--open"')
-      }
-      return true
-    })
+    // .check((argv) => {
+    //   if (argv.build && argv.smokeTest) {
+    //     throw new Error('Can not provide both "--build" and "--smoke-test"')
+    //   }
+    //   if (argv.build && argv.open) {
+    //     throw new Error('Can not provide both "--build" or "--open"')
+    //   }
+    //   return true
+    // })
+    .epilogue(
+      `Also see the ${terminalLink(
+        'Redwood CLI Reference',
+        'https://redwoodjs.com/reference/command-line-interface#storybook'
+      )}`
+    )
 }
 
 export const handler = ({
-  open,
-  port,
-  build,
-  buildDirectory,
-  managerCache,
-  smokeTest,
+  open = true,
+  port = 7910,
+  build = false,
+  buildDirectory = 'public/storybook',
+  managerCache = true,
+  smokeTest = false,
```

Comment:
```suggestion
  open,
   port,
   build,
   buildDirectory,
   managerCache,
   smokeTest,
```

Ah, this makes sense. But what you probably don't know is that Yargs will handle all this via the defaults set in the build() Lines 18-49. So this can be reverted.